arch/arm/boot/dts/overlays/revpi-flat-s-2022-overlay.dts

```diff
@@ -0,0 +1,367 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Flat S (2022)
+ */
+
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/{
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-flat-s-2022",
+				     "kunbus,revpi-flat", "brcm,bcm2711";
+
+			aout_vref: fixedregulator_2v5x4 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x4_amplified";
+				regulator-min-microvolt = <10000000>;
+				regulator-max-microvolt = <10000000>;
+				regulator-always-on;
+			};
+
+			ain_vref: fixedregulator_2v5x5 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x5_amplified";
+				regulator-min-microvolt = <12500000>;
+				regulator-max-microvolt = <12500000>;
+				regulator-always-on;
+			};
+
+			wl_on: reg_wl {
```

Comment:
As the signal is no longer called `wl_on` it would be best to rename the regulator node as well. In the cm4 dts they use the name wifi_pwrseq which would describe it well and is independent from the signal name.